### DISCUSSION TITLE
Disable the wireless card at pineapple startup when the buttons do not work

### DISCUSSION THREAD

**Discussion by kaarloss** [xchwarze/wifi-pineapple-cloner#9] [Q&A] [answered]
Thanks for the info. I have tried it with a tp-link archer ac1750 c7 v2, when starting the pineapple, the wireless switch and the reset button do not deactivate the wireless card and I cannot continue for configure . Ssh no work.

Does anyone know, how I can continue to change the root password and work with pineapple?

**Comment by xchwarze** [comment, marked as answer]
The firmware as mentioned in the download repo notes has a default password. Also the mentioned defect should never happen either.
Did you install following the steps?

**Reply by kaarloss** [reply]
I'm sorry for wasting your time, I actually made a mistake in one step, it works very well. Thank you very much Xchwarze.

**Reply by rjthecarguy** [reply]
I m having the same issue.  I followed the install instructions but it stays on the "Secure Setup" page and keeps asking me to turn off the WiFi.

Turning off the Wifi on my computer has no effect.

Can you please share what step you missed and then fixed to avoid the error?

Thanks!

**Reply by xchwarze** [reply]
As I confirm in the following message of kaarloss the problem is due to not following the installation steps.
That this happens is a sign of a bad installation.

**Comment by kaarloss** [comment]
Install following the steps from schwarze, choose the proper firmware, not
generic, it was my error, works for me

**Comment by kuno23** [comment]
Hi i compiled fw by myself with the automatic build step because there is no fw for Mikrotik RB951UI2HnD, and im stuck on the screen Disabling WiFi is recommended for initial setup. and the reset button do nothing. Any suggestion.?

**Reply by xchwarze** [reply]
see this https://github.com/xchwarze/wifi-pineapple-cloner/blob/master/tools/fs-patcher.sh#L144
```bash
cat /etc/rc.button/wps
```

you should see this block in the file:
https://github.com/xchwarze/wifi-pineapple-cloner/blob/master/files/common/etc/wps#L41

if to platform executes the action from another custom button then simply run this
```bash
touch /tmp/button_setup
```

**Reply by kuno23** [reply]
Hi thanks for quick reply, the touch command do the trick, but another question. My programing skills are not so good, can say that almost none, so it is possible to edit some file that after compiling the fw it will work as expected that the reset button (because there is no wps button on the device) will do the function of disabling wifi and the setup will continue.?
